tests/helpers/mod.rs

```diff
@@ -140,3 +140,52 @@ macro_rules! test_lint_diagnostics {
       ::insta::assert_snapshot!(formatted_diags, @$expected_diagnostics);
   }};
 }
+
+use cairo_lang_filesystem::db::CrateSettings;
+use cairo_lang_utils::Intern;
+
+use crate::CRATE_CONFIG;
```

Comment:
Could we move those to the top of the file?

Reply:
or even create a separate file containing this single function in the `helpers` module